python/cuml/tests/conftest.py

```diff
@@ -186,6 +190,18 @@ def pytest_configure(config):
         hypothesis.settings.load_profile("unit")
 
 
+def pytest_pyfunc_call(pyfuncitem):
+    """Skip tests that require the cudf.pandas accelerator
+
+    Tests marked with `@pytest.mark.cudf_pandas` will only be run if the
+    cudf.pandas accelerator is enabled via the `cudf.pandas` plugin.
+    """
+    cudf_pandas_enabled = "ModuleAccelerator" in str(pd)
```

Comment:
I would recommend checking `cudf.pandas.LOADED`

Reply:
+1

Reply:
Nice. I like it.

I found https://docs.rapids.ai/api/cudf/stable/cudf_pandas/faq/#how-can-i-tell-if-cudf-pandas-is-active which suggests looking at the string representation. Maybe worth updating.